### PR TITLE
Followup to PR 9088 comments

### PR DESCRIPTION
**Pull request by cwharris**
These are some minor updates requested for PR #9088 that I forgot to push prior to merging the PR.

### CODE REVIEW

File: cpp/include/cudf/io/detail/json.hpp [hunk 1]
```diff
@@ -12,12 +12,7 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- */
-
-/**
- * @file json.hpp
- * @brief cuDF-IO reader classes API
- */
+ */\
```
Comment (bdice) [inline cpp/include/cudf/io/detail/json.hpp:15]: This extra backslash seems like a typo?
```suggestion
 */
```

File: cpp/src/io/json/reader_impl.cu [hunk 2]
```diff
@@ -345,7 +345,7 @@ std::pair<std::vector<std::string>, col_map_ptr_type> get_column_names_and_map(
   rmm::cuda_stream_view stream)
 {
   // If file only contains one row, use the file size for the row size
-  uint64_t first_row_len = d_data.size() / sizeof(char);
+  uint64_t first_row_len = d_data.size();
```
Comment (bdice) [inline cpp/src/io/json/reader_impl.cu:348]: Is this value ever used? It looks like the next 20-ish lines always overwrite whatever value is stored here. If that's the case, why not just declare it without a value? Am I missing something?

**Review by bdice — CHANGES_REQUESTED** [review]

File: cpp/src/io/json/reader_impl.cu (hunk 2 above)
Comment (bdice) [reply cpp/src/io/json/reader_impl.cu:348]: This looks fine after all.

**Review by bdice — APPROVED** [review]

**Review by ttnghia — APPROVED** [review]

**Comment by cwharris** [comment]
@gpucibot merge

**Pull request merged**